.github/workflows/autotest.yml

```
@@ -21,6 +21,18 @@ jobs:
       - name: Run hatch test
         shell: bash
         run: hatch test
+  formattest:
+    name: Code Formatting check
```

```suggestion
    name: Code formatting check
```
Also add a name for the pytest tests job.